mobile/lib/shared/services/immich_logger.service.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:flutter/foundation.dart';
+import 'package:hive/hive.dart';
+import 'package:immich_mobile/constants/hive_box.dart';
+import 'package:immich_mobile/shared/models/in_app_logger_message.model.dart';
+
+class ImmichLogger {
+  String logContext = "";
```

Comment:
You could remove the default value and make the field final (it's required in the constructor)

`final String logContext;`